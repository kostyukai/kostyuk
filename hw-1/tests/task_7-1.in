-157.1215624
