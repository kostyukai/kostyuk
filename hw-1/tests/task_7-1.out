11000000000001101001110100011111000111101011011010100100011011001010010001101100
